atable/src/components/ARow.vue

```diff
@@ -43,6 +44,20 @@ const tableData = inject<TableDataStore>(props.tableid)
 const rowEl = ref<HTMLTableRowElement>(null)
 const numberedRowWidth = tableData.numberedRowWidth.value
 
+const { x, y, style } = useDraggable(rowEl, {
+	initialValue: { x: 5, y: (props.rowIndex + 1) * 21 },
```

Comment:
Don't use [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)). Is the `21` supposed to be the row height? Can we compute it instead?